apps/leave-app/webapp/src/view/LeaveHistory/component/LeaveCard.tsx

```diff
@@ -0,0 +1,143 @@
+// Copyright (c) 2025 WSO2 LLC. (https://www.wso2.com).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import { Box, Card, CardContent, Stack, Typography, useTheme } from "@mui/material";
+
+export interface LeaveCardProps {
+  id: string;
+  type: string;
+  startDate: string;
+  endDate: string;
+  duration: string;
+  status: "approved" | "pending" | "rejected";
+  month: string;
+  day: string;
+}
+
+export default function LeaveCard(leave: LeaveCardProps) {
```

Comment:
Similar to what you have done with other components, and since you only need duration, shall we destructure the props as I have suggested below?

```suggestion
export default function LeaveCard({duration}: LeaveCardProps) {
```